common/templates/_deployment.yaml

```diff
@@ -34,6 +35,29 @@ spec:
     {{- if (coalesce .Values.priorityClassName .Values.global.priorityClassName) }}
       priorityClassName: {{ (coalesce .Values.priorityClassName .Values.global.priorityClassName) }}
     {{- end }}
+    {{- if has .Chart.Name (values .Values.global.HelmNames.InternalService) }}
```

Comment:
can we add a way to individually disable this? 
eg
`     {{- if and ( has .Chart.Name (values .Values.global.HelmNames.InternalService) chart) .Values.enableDacpacInitContainer }} `

and add ` enableDacpacInitContainer: true` to the values.yaml

Reply:
added disabling part and modified the values.yaml file